tests/test_geometry.py

```diff
@@ -229,8 +230,9 @@ def test_truetype():
 
 @pytest.mark.skipif(sys.version_info < (3, 0), reason="unicode test requires python3")
 def test_unicode():
-    _ = pytest.importorskip(
-        "phidl.font", reason="Testing of ttf/otf fonts requires the freetype package."
+    pytest.importorskip(
+        "phidl.font", reason="Testing of ttf/otf fonts requires the freetype package.",
+        exc_type=ImportError,
```

Comment:
This could go in a separate PR, and this might not be the correct solution.
https://docs.pytest.org/en/8.2.x/deprecations.html#import-or-skip-import-error